dev-client/src/components/FeatureFlagControl.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+import {useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Divider, Switch} from 'react-native-paper';
+
+import {
+  Heading,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {APP_CONFIG, MMKV} from 'terraso-mobile-client/config';
+
+export const FeatureFlagControl = () => {
+  // It may cause issues to toggle a feature flag state while the app is running,
+  // so expect the user to restart the app if they want to change the feature flag state.
+  // This should never be exposed in production.
+
+  const currentFlagState = APP_CONFIG.FF_offline;
```

Comment:
I don't think there's a standard. I don't expect us to have many feature flags, so I chose to keep it short, since we will use it in every place where we want to branch behavior. But if we were planning to have several flags I might opt for more description. It should also be pretty easy to change in the future if we end up wanting to change its name.

Reply:
Actually, I'll plan to add some description of what the flag is